integration/test_grid_manager.py

```diff
@@ -0,0 +1,182 @@
+import sys
+import time
+import json
+import shutil
+from os import mkdir, unlink, listdir, utime
+from os.path import join, exists, getmtime
+
+from allmydata.util import keyutil
+from allmydata.util import base32
+from allmydata.util import configutil
+
+import util
+
+import pytest_twisted
+
+
+@pytest_twisted.inlineCallbacks
+def test_create_certificate(reactor):
+    gm_config = yield util.run_tahoe(
+        reactor, "grid-manager", "--config", "-", "create",
+    )
+    privkey_bytes = json.loads(gm_config)['private_key'].encode('ascii')
+    privkey, pubkey_bytes = keyutil.parse_privkey(privkey_bytes)
+    pubkey = keyutil.parse_pubkey(pubkey_bytes)
+
+    gm_config = yield util.run_tahoe(
+        reactor, "grid-manager", "--config", "-", "add",
+        "alice", "pub-v0-kzug3ut2m7ziihf3ndpqlquuxeie4foyl36wn54myqc4wmiwe4ga",
```

Comment:
This is just testing the certificate-creation process, so the public key here doesn't matter because we're only confirming (lower down) that the grid-manager has indeed signed the certificate.